app/navigation/Navigator.scala

```diff
@@ -34,7 +34,7 @@ class RootNavigator @Inject()() extends Navigator {
         case WhatYouWillNeedPage(srn) => {
           val isDataEmpty = userAnswers.data.decryptedValue == JsObject.empty
           if (isDataEmpty) {
-            routes.IndexController.onPageLoad
+            routes.WhichTaxYearController.onPageLoad(srn, NormalMode)
```

Comment:
```
Scenario 2 - Constructing pensions scheme return date
Given that the user has selected a scheme from the PSA\PSP dashboard (design pending)
when constructing the date in the title 
then ensure that the notice to file date is carried through and populated in the title
[Refer [Content page](https://docs.google.com/document/d/1p3xQehCoBAjvQVtIcWG8IrVFtaQyMQqqX0FpOiumAJA/edit?usp=drive_link) Notes section]
```
In the absence of a NTF date being "passed" I think it is sensible to navigate here first if there is no data/date available